runtime/scripts/exam.js

```diff
@@ -737,30 +737,6 @@ Exam.prototype = /** @lends Numbas.Exam.prototype */ {
         }
     },
 
-    /** Show the question menu.
-     * @fires Numbas.Exam#event:showMenu
-     */
-    showMenu: function() {
```

Comment:
`showMenu` is still needed: it's called from [`Numbas.controls.backToMenu`](https://github.com/numbas/Numbas/blob/master/runtime/scripts/controls.js#L67).